addons/web_editor/static/src/js/wysiwyg/widgets/alt_dialog.js

```diff
@@ -2,16 +2,16 @@ odoo.define('wysiwyg.widgets.AltDialog', function (require) {
 'use strict';
 
 var core = require('web.core');
-var Dialog = require('wysiwyg.widgets.Dialog');
+var WeDialog = require('wysiwyg.widgets.Dialog');
```

Comment:
It's not the most important commit but this would have saved me some time and confusion, so I assume the same can be said for others.

I don't think we should avoid creating a conflict, if we consider it a good change it should be done. But why do you not agree with it?

Reply:
If the change is not really useful, of course we should avoid creating conflicts and adding diff and history to the code. I am ok with your code, I am just not agreeing that we should create a commit to modify this. People can name their import however they want, and it is not breaking any convention.

Reply:
I'm not convinced but I don't care enough about this change to debate for it, so I'll drop this commit.